apps/shop/serializers.py

```diff
@@ -18,6 +18,16 @@ class Meta:
 class OrderLineSerializer(serializers.ModelSerializer):
     orders = OrderSerializer(many=True)
 
+    def validate_orders(self, orders: dict):
+        for order in orders:
+            item = Item.objects.get(pk=order.get("object_id"))
+            if not item.available:
+                raise serializers.ValidationError(
+                    "Enklte av de gitte produktene er ikke tilgjengelig"
```

Comment:
Why is this error-message in Norwegian, while the one in `OrderLine.clean()` was in English?

Reply:
The validation errors are potentially user-facing, and are mostly written in Norwegian in this project. I just cleared up the anomaly.